lib/types/orders.ts

```diff
@@ -64,6 +64,8 @@ export type OrderPortion = OrderIdentifier & {
   localId?: string;
 };
 
+export type OrderInvalidation = OrderPortion & Remote;
```

Comment:
I don't think we're using this type anywhere so we can drop it for now.

Reply:
right, it's from a previous iteration of mine.